src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -82,18 +88,85 @@ public string LanguageVersion
             }
         }
 
-        [Option(
-            "-o|--output <dir>",
-            CommandOptionType.SingleValue,
-            Description = "The destination directory.")]
         public string DestinationDir { get; set; }
 
-        [Option(
-            OptionTemplates.ManifestDir,
-            CommandOptionType.SingleValue,
-            Description = "The path to a directory into which the build manifest file should be written.")]
         public string ManifestDir { get; set; }
 
+        public static Command Export()
+        {
+            var logOption = new Option<string>(OptionTemplates.Log, OptionTemplates.LogDescription);
+            var debugOption = new Option<bool>(OptionTemplates.Debug, OptionTemplates.DebugDescription);
+            var sourceDirArgument = new Argument<string>("SourceDir", "The source directory.");
+            var platformOption = new Option<string>(OptionTemplates.Platform, OptionTemplates.PlatformDescription);
+            var platformVersionOption = new Option<string>(OptionTemplates.PlatformVersion, OptionTemplates.PlatformVersionDescription);
+            var packageOption = new Option<bool>(OptionTemplates.Package, OptionTemplates.PackageDescription);
+            var osReqOption = new Option<string>(OptionTemplates.OsRequirements, OptionTemplates.OsRequirementsDescription);
+            var appTypeOption = new Option<string>(OptionTemplates.AppType, OptionTemplates.AppTypeDescription);
+            var buildCommandFileNameOption = new Option<string>(OptionTemplates.BuildCommandsFileName, OptionTemplates.BuildCommandsFileNameDescription);
+            var compressDestDirOption = new Option<bool>(OptionTemplates.CompressDestinationDir, OptionTemplates.CompressDestinationDirDescription);
+            var propertyOption = new Option<string[]>(aliases: new[] { "-p", OptionTemplates.Property }, OptionTemplates.PropertyDescription);
```

Comment:
nit: it may be worth to change `OptionTemplates.Property` to `OptionTemplates.PropertyAliases` and have it return a `string[]` that is the array of values accepted as you have on this line

```cs
public string[] PropertyAliases = new string[] { "-p", "--property" };
```

This comment applies for the other alias values we have.